addons/website/static/src/js/content/website_root.js

```diff
@@ -82,6 +91,30 @@ var WebsiteRoot = publicRootData.PublicRoot.extend({
         }
         return registry;
     },
+    /**
+     * Toggles the fullscreen mode.
+     *
+     * @private
+     * @param {Boolean} state toggle fullscreen on/off (true/false)
+     */
+    async _toggleFullscreen(state) {
+        this.isFullscreen = state;
+        document.body.classList.toggle('o_fullscreen', this.isFullscreen);
+        let resizing = true;
+        const resizeFunction = () => {
+            if (resizing) {
+                window.dispatchEvent(new Event('resize'));
+                window.requestAnimationFrame(resizeFunction);
+            }
+        };
+        window.requestAnimationFrame(resizeFunction);
+        document.body.addEventListener('transitionend', async ev => {
+            if (ev.target.matches('body') && ev.propertyName === 'padding-top') {
```

Comment:
Feels like `ev.target === document.body` would be a better way to write the condition